<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AMCP - Automated Measurement of Crystal Parameters</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Overview</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Crystal Specs</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="3" column="2">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>[Hz]</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>[Hz]</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>[Hz]</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;F&lt;span style=&quot; vertical-align:sub;&quot;&gt;center&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="f_center">
             <property name="text">
              <string>26E6</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;F&lt;span style=&quot; vertical-align:sub;&quot;&gt;min&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="f_min">
             <property name="text">
              <string>25E6</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;F&lt;span style=&quot; vertical-align:sub;&quot;&gt;max&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="f_max">
             <property name="text">
              <string>27E6</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Select VNA</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="select_vna">
             <item>
              <property name="text">
               <string>MiniVNA</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NanoVNA</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QPushButton" name="btn_run_measurement">
            <property name="text">
             <string>Run Measurement</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btn_run_estimation">
            <property name="text">
             <string>Run Estimation</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Results</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>L1:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" alignment="Qt::AlignLeft">
            <widget class="QLabel" name="L1_res">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2" alignment="Qt::AlignLeft">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>[mH]</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>R1:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2" alignment="Qt::AlignLeft">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>[Î©]</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>C0:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" alignment="Qt::AlignLeft">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>[fF]</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>C1:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" alignment="Qt::AlignLeft">
            <widget class="QLabel" name="C0_res">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2" alignment="Qt::AlignLeft">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>[pF]</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" alignment="Qt::AlignLeft">
            <widget class="QLabel" name="R1_res">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="C1_res">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>160</height>
           </size>
          </property>
          <property name="title">
           <string>Crystal Model</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="model">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/crystal/images/xtal_model.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="margin">
              <number>5</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" rowspan="3" colspan="3">
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="PlotWidget" name="graphWidget" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Setup</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>Python</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="1">
            <widget class="QLineEdit" name="python3_path"/>
           </item>
           <item row="1" column="0" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_24">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Python 3 Interpreter:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QToolButton" name="toolButton_5">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>MiniVNA</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_6"/>
           </item>
           <item row="3" column="2">
            <widget class="QToolButton" name="toolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item row="1" column="2">
            <widget class="QToolButton" name="toolButton_3">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_20">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Calibration File:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>VnaJ</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>NanoVNA</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_7"/>
           </item>
           <item row="2" column="0" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_22">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>NanoVNA wrapper:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QToolButton" name="toolButton_4">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Calibration File:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_5"/>
           </item>
           <item row="3" column="2">
            <widget class="QToolButton" name="toolButton_2">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPlainTextEdit" name="logfile">
          <property name="plainText">
           <string>start log:
--------------------
</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>1</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionTest"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_run_measurement</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>run_measurement()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>683</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>694</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_run_estimation</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>run_estimation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>652</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>run_measurement()</slot>
  <slot>run_estimation()</slot>
 </slots>
</ui>
